pegasus/sites.v3/code.org/views/interactive_map.haml

```diff
@@ -35,6 +35,9 @@
     %strong= format_integer_with_commas(jobs_nationwide)
     Open computing jobs nationwide
 
+    %strong=number_to_percentage(47, precision: 0)
```

Comment:
Ah, do we plan to update this number somewhat regularly?

Reply:
We plan to have engineering update this number every Fall